app/src/components/Modal.js

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import { Transition } from "react-transition-group";
+import Modal from "@mui/joy/Modal";
```

Comment:
Why not https://mui.com/material-ui/react-dialog/ ? 
To be consistent, it's good to stick with one mui repo, `material` or `joy`, not both :)